lib/compat/wordpress-6.5/class-wp-navigation-block-renderer.php

```diff
@@ -547,20 +547,28 @@ private static function get_nav_element_directives( $should_load_view_script ) {
 	 */
 	private static function handle_view_script_loading( $attributes, $block, $inner_blocks ) {
 		$should_load_view_script = static::should_load_view_script( $attributes, $inner_blocks );
+		$is_gutenberg_plugin     = defined( 'IS_GUTENBERG_PLUGIN' ) && IS_GUTENBERG_PLUGIN;
```

Comment:
As far as I know, this class needs to be backported to WordPress. I saw @youknowriad and @tellthemachines discussing options, including moving it to the block folder to automatically sync it from npm to WP core. In that case, the check would be necessary.

Reply:
Let's keep it for now, then. Thanks, Greg.

Reply:
Might be good to keep this in even if the file isn't auto-synced so that it's clear which parts of the logic should be added to core. 

Is this switch to modules planned for 6.5, or will it stay experimental for a while yet?

Reply:
We don't know yet. But if the code is ready and there's consensus on the API, it could be 🙂